wordpress-mobile/WordPress-Android
# Issue/6509 email not found

#### Note
This should be merged ***after*** https://github.com/wordpress-mobile/WordPress-Android/pull/6671.

### Fix
Add connected, email not found screen as described in https://github.com/wordpress-mobile/WordPress-Android/issues/6509.

### Test
0. Clear app data.
1. Tap ***Log In*** button.
2. Tap Google login view.
3. Choose Google account with email address not used for WordPress account.
4. Notice interface shown below.
![auth_i5_06_connected_email_not_found](https://user-images.githubusercontent.com/3827611/29892078-7b28e098-8d8a-11e7-916a-cde96dff2cc2.png)
